Code/Tools/ProjectManager/Source/CreateProjectCtrl.cpp

```diff
@@ -233,15 +233,16 @@ namespace O3DE::ProjectManager
     {
         if (m_stack->currentWidget() == m_newProjectSettingsScreen)
         {
-            return m_newProjectSettingsScreen->Validate();
+            return m_newProjectSettingsScreen->Validate().IsSuccess();
         }
 
         return true;
     }
 
     void CreateProjectCtrl::CreateProject()
     {
-        if (m_newProjectSettingsScreen->Validate())
+        AZ::Outcome<void, QString> settingsValidation = m_newProjectSettingsScreen->Validate();
+        if (settingsValidation.IsSuccess())
```

Comment:
`IsSuccess()` is clearer, but [AZ::Outcome overrides the bool()](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Outcome/Internal/OutcomeImpl.h#L237) operator to return IsSuccess() so you could just use `if(settingsValidation)`

Reply:
Not sure I agree from a code reading standpoint. I prefer verbosity, especially if I'm reading code late at night. Makes it easier to gauge the intended logic.

Reply:
Yes, I started out as just `if(settingsValidation)` then added the `IsSuccess()` for clarity. I'm happy with this either way though if coding guidelines prefer one over the other - though I also did a quick search through o3de in general and is seems IsSuccess is used more often than without.